Added docs contents for caver-js v1.9.0 (#456)
klaytn/klaytn-docs: docs/dapp/sdk/caver-js/api-references/caver.rpc/klay.md

| blockNumber | string | Block number where this transaction was in. |
| codeFormat | string | (optional) The code format of smart contract code. |
| contractAddress | string | The contract address created, if the transaction was a contract creation, otherwise `null`. |

> | effectiveGasPrice | string | The actual value per gas deducted from the senders account. Before Magma hard fork, this is equal to the transaction’s gas price. After, it is equal to baseFee of the block header. |

Reviewer: ```suggestion
| effectiveGasPrice | string | The actual value per gas deducted from the sender. Before Magma hard fork, this is equal to the transaction’s gas price. After Magma, it is equal to the value of baseFee in the block header. |
```

> | effectiveGasPrice | string | The actual value per gas deducted from the sender. Before Magma hard fork, this is equal to the transaction’s gas price. After Magma, it is equal to the value of baseFee in the block header. |

Reviewer 2: ```suggestion
| effectiveGasPrice | string | The actual value per gas deducted from the sender. Before the Magma hard fork, this value was equal to the transaction’s gas price. After the Magma hard fork, it is equal to the value of `baseFee` in the block header. |
```

| typeInt | number | An integer representing the type of the transaction. |
| value | string | Value transferred in peb. |

**NOTE** `effectiveGasPrice` is supported since caver-js [v1.9.0](https://www.npmjs.com/package/caver-js/v/1.9.0).

**Example**

```javascript

> // Before Magma hard fork

Reviewer 2: ```suggestion
// Before the Magma hard fork
```

    typeInt: 8,
    value: '0x8ac7230489e80000',
}

> // After Magma hard fork

Reviewer 2: ```suggestion
// After the Magma hard fork
```
